Gemfile

```diff
@@ -33,7 +33,7 @@ end
 
 
 group :development, :test do
-  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'pry'
```

Comment:
`pry-rails` instead of `pry` will automatically start `rails console` with Pry ☝️